README.md

```diff
@@ -100,7 +100,7 @@ $ curl -s https://packagecloud.io/install/repositories/intelsdi-x/snap/script.de
 $ sudo apt-get install -y snap-telemetry
 ```
 
-Linux Mint 17/17.* (for Linux Mint 18/18.* use `dist=xenial`)
+For distros that are compatible with RedHat/Ubuntu packages, use the environment variable `os=` and `dist=` to override the OS detection script. For example Linux Mint 17/17.* (use `dist=xenial` for Linux Mint 18/18.*):
```

Comment:
Do you think we need to note we don't specifically test against these other flavors of Linux so YMMV? Or is that clearly enough noted?